Packs/Threat_Crowd/Integrations/ThreatCrowd_v2/ThreatCrowd_v2.py

```diff
@@ -73,6 +74,9 @@ def ip_command(client: Client, args: Dict[str, Any]) -> List[CommandResults]:
     ips = argToList(args.get('ip'))
     for ip in ips:
         res = client._http_request(method='GET', url_suffix=api_url, params={'ip': ip})
+        if res.get('response_code') == IP_NO_DATA_RESPONSE_CODE:
+            # Indicating no data was found regarding the requested IP. Skipping.
+            continue
```

Comment:
If I do for example `!ip ip=8.8.8.8` and it doesn't have reputation in the response, will I get an output?

Reply:
res = {'response_code': 0}